src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,30 +5,16 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> findHappyNumbersInRange(long start, long end) {
```

Comment:
HappyNumbers is an existing term with different meaning.

Reply:
Name confuses, as said by Tobias